addons/mail/static/tests/message/message.test.js

```diff
@@ -652,6 +652,7 @@ test("Reaction summary", async () => {
                 after: ["span", { textContent: "Smileys & Emotion" }],
                 text: "😅",
             });
+            await contains(".o-mail-MessageReaction", { text: `😅${idx + 1}` });
```

Comment:
What is necessary diff to pass the test? Or just to make it more robust?

Reply:
Depends on how the bus notification is received, it could happen that the reaction is not inserted before the hover. So test fails on not finding the right summary on hover.

Reply:
That would mean the `.o-mail-MessageReactionList-preview` is not properly reactive?
Otherwise it should update itself when the data is received, even if it's not already open.